rubin_sim/maf/metrics/incremental_template_metric.py

```diff
@@ -102,7 +111,13 @@ def run(self, data_slice, slice_point=None):
         idx_template_inputs = ok_template_input[: self.n_images_in_template]  # Images included in template
 
         Night_template_created = data_slice[self.night_col][idx_template_created]
```

Comment:
While we're here -- this seems wrong, with a capital letter at the start of the variable?